packages/vue/src/examples/pages/static/Static.vue

```diff
@@ -60,18 +72,46 @@
         information collected offline by Luma. All Luma websites are covered by
         this privacy policy.
       </SfContentPage>
+      <SfContentPage title="Delivery">
+        This website ("website") is operated by Luma Inc., which includes Luma
```

Comment:
![Zrzut ekranu 2020-04-1 o 14 11 59](https://user-images.githubusercontent.com/12138170/78136113-5fe8ef80-7423-11ea-9391-8f53590d0e4f.png)
maybe we can add padding: 0 var(--spacer-sm) ?